glide.yaml

```diff
@@ -40,4 +43,9 @@ import:
   version: d4647c9c7a84d847478d890b816b7d8b62b0b279
 - package: github.com/posener/complete
   version: v1.1.2
+- package: github.com/openshift/origin
```

Comment:
shouldn't `github.com/openshift/origin` and `github.com/openshift/client-go` versions match?

Reply:
we are using just one package from origin, why not just import that subpackage?

Reply:
@kadel , https://github.com/openshift/client-go/releases don't release regularly, so I took it from the master then :(

Reply:
openshift directly uses `master` from `openshift/client-go`
ref: https://github.com/openshift/origin/blob/master/glide.yaml#L42
shall I use the same ?

Reply:
I created issue for the same: https://github.com/openshift/client-go/issues/66

Reply:
only origin master uses client-go master. If you check glide.yaml for 3.11 you can see that they are using client-go 3.11 https://github.com/openshift/origin/blob/v3.11.0/glide.yaml#L43